### DISCUSSION TITLE
APP: Adding filter/search via URL

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#13892] [Feature Requests]
It would be great if we could pass filters to the App via URL. The schema could be identical to the API.

I think the biggest pain point to discuss would be if the URL should always be synced with the filters. That could be an unneeded overload for most users. As a mvp they could be parsed on initial pageload and than be stripped out of the URL 🤔

In that case probably a button  like "create sharable link" would make sense.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
